src/core/ext/filters/client_channel/resolver/polling_resolver.h

```diff
@@ -69,6 +69,7 @@ class PollingResolver : public Resolver {
   const std::string& name_to_resolve() const { return name_to_resolve_; }
   grpc_pollset_set* interested_parties() const { return interested_parties_; }
   const ChannelArgs& channel_args() const { return channel_args_; }
+  std::shared_ptr<WorkSerializer> work_serializer() { return work_serializer_; }
```

Comment:
I think this should return a raw pointer, not a `shared_ptr<>`.  The subclass is not going to need to take its own ref.

Reply:
Sounds good, the raw pointer works for my use case.